server/apps/publish/publish_filters/filter_condition.py

```diff
@@ -0,0 +1,150 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+import logging
+import re
+from superdesk.resource import Resource
+from superdesk.services import BaseService
+
+logger = logging.getLogger(__name__)
+
+
+class FilterConditionResource(Resource):
+    schema = {
```

Comment:
There's a function in place to check the same filter is used before. Also the name needs to be unique.